sound/soc/sof/intel/hda-dsp.c

```diff
@@ -1059,12 +1062,10 @@ int hda_dsp_suspend(struct snd_sof_dev *sdev, u32 target_state)
 
 	/* stop hda controller and power dsp off */
 	ret = hda_suspend(sdev, false);
-	if (ret < 0) {
+	if (ret < 0)
 		dev_err(bus->dev, "error: suspending dsp\n");
```

Comment:
while here, can we drop the `error:` from the print (and print out the function name)?